### PR TITLE
Use the current branch test-utils to build cctools

### PR DESCRIPTION
**Pull request by Felixoid**
Use a proper tag to build cctools

### CODE REVIEW

File: docker/packager/binary/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # docker build -t clickhouse/binary-builder .
 ARG FROM_TAG=latest
-FROM clickhouse/test-util:latest AS cctools
+FROM clickhouse/test-util:54187 AS cctools
```
Comment (alexey-milovidov) [inline docker/packager/binary/Dockerfile:3]: Needs a comment to tell what `54187` means and why it is needed.

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit 85ca14d78ee17b8c66997ac4380b12720ad0bd71 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/61276/85ca14d78ee17b8c66997ac4380b12720ad0bd71/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Push to Dockerhub</td><td>The check for building and pushing the CI related docker images to docker hub</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

**Comment by Felixoid** [comment]
https://github.com/ClickHouse/ClickHouse/actions/runs/8263087015/job/22604602461#step:6:102 docker is a Hub 503 error

**Comment by Felixoid** [comment]
Ok, it works. Let's fix the release

**Pull request merged**